pkg/shell/terraform.go

```diff
@@ -176,6 +181,38 @@ func ExportOutputs(tf *tfexec.Terraform, metadataFile string, artifactsDir strin
 // ImportInputs will search artifactsDir for files produced by ExportOutputs and
 // combine/filter them for the input values needed by the group in the Terraform
 // working directory
-func ImportInputs(tf *tfexec.Terraform, metadataFile string, artifactsDir string) error {
+func ImportInputs(workingDir, metadataFile string, artifactsDir string) error {
+	deploymentRoot := path.Clean(path.Join(workingDir, ".."))
+	thisGroup := path.Base(workingDir)
+
+	outputsByGroup, err := getOutputsFromEarlierGroups(thisGroup, metadataFile)
+	if err != nil {
+		return err
+	}
+
+	if _, err = GetDeploymentKinds(metadataFile, deploymentRoot); err != nil {
+		return err
+	}
+
+	allAttributes := make(map[string]cty.Value)
+	for group, outputs := range outputsByGroup {
+		if len(outputs) == 0 {
+			continue
+		}
+		filepath := outputsFile(artifactsDir, group)
+		attrs, err := modulereader.ReadHclAttributes(filepath)
+		if err != nil {
+			return fmt.Errorf("could not load file %s; consider running \"ghpc export-outputs %s/%s\".\n%v", filepath, deploymentRoot, group, err)
```

Comment:
@nick-stroud I realize this is probably a good candidate for `TfError`. Will update after review.